watson-discovery/watson-discovery.lua

```diff
@@ -0,0 +1,94 @@
+--[[
+# [Watson API Explorer]https://console.bluemix.net/docs/services/discovery/getting-started-tool.html#getting-started-with-the-tooling) connector.
+
+[Discovery](https://watson-api-explorer.mybluemix.net/apis/discovery-v1#!/Configurations/listConfigurations)
+
+## Input parameters
+* watson_domain - e.g. `https://watson-api-explorer.mybluemix.net/`.
+* api_endpoint - API endpoint URI, e.g. `personality-insights/api/v3/profile`.
```

Comment:
the example is bogus, it should come from this list
https://watson-api-explorer.mybluemix.net/apis/discovery-v1
e.g. `v1/environments/system/configurations`